docs/guide/bundles.md

```diff
@@ -24,20 +24,19 @@ To learn more about organizing your eZ Platform project, see [Best Practices](be
 
 You can generate a new bundle using a `generate:bundle` command. See [Symfony documentation on generating bundles](http://symfony.com/doc/current/bundles/SensioGeneratorBundle/commands/generate_bundle.html).
 
-#### How to remove a bundle?
+### How to remove bundles?
 
 To remove a bundle (either one you created yourself, or an out-of-the-box one that you do not need) see the [How to Remove a Bundle](http://symfony.com/doc/current/bundles/remove.html) instruction in Symfony doc.
 
 ## Built-in bundles
 
-Some of the key built-in bundles are:
-
-#### eZ Platform kernel
+The eZ Platform root directory contains multiple sub-directories.
```

Comment:
A bit misleading. In an installation the repos are stored in `vendor/ezsystems`, not the root itself.